experimental/dds/tree2/src/test/snapshots/testTrees.ts

```diff
@@ -46,22 +64,28 @@ function generateTreeRecursively(
 		const field = tree.editor.sequenceField(fieldUpPath);
 
 		for (let i = 0; i < nodesPerField; i++) {
-			const writeCursor = singleTextCursor({
-				type: brand("TestValue"),
-				value: currentValue.toString,
-			});
-			field.insert(i, writeCursor);
+			if (height === 1) {
+				const writeCursor = singleTextCursor({
+					type: leafNodeSchema.name,
+					value: currentValue.toString,
+				});
+				field.insert(i, writeCursor);
+				currentValue.value++;
+			} else {
+				const writeCursor = singleTextCursor({
+					type: rootNodeSchema.name,
+				});
+				field.insert(i, writeCursor);
 
-			currentValue.value++;
```

Comment:
Looks like you're no longer incrementing `currentValue.value`. Is that intentional?

Reply:
At least not in the `height !== 1` code path.

Reply:
Yes, this is intended: I figured having the leaves in the tree use all the numbers made more sense than only using a subset of the numbers (skipping on for each interior node.

Reply:
Agreed, and we're not using that number to count anything on the outside, so there's no harm.